docs/data/system/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -37,13 +37,27 @@ You can follow progress on https://github.com/mui/material-ui/issues/30671.
 
 ### Grid
 
-The `Grid` component was updated to match the API of the new `PigmentGrid` component, to allow interoperability between the two:
+The `Grid` component was stabilized and updated to match the API of the new `PigmentGrid` component, to allow interoperability between the two:
```

Comment:
Can we link `PigmentGrid` somewhere?

Reply:
I see how people might think: "wtf is `PigmentGrid`?"

Reply:
We could remove the mention of `PigmentGrid`; I agree it's confusing. What do you think?

Reply:
👍

Reply:
Updated 👌🏼